### PR TITLE
Use let!(:var) instead of let(:var); var

### PR DESCRIPTION
**Pull request by benthorner**

### CODE REVIEW

File: spec/features/edit_dataset_spec.rb [hunk 1]
```diff
@@ -19,8 +19,6 @@
                                                  owner: user ) }
 
   before(:each) do
-    allow_any_instance_of(UrlValidator).to receive(:validPath?).and_return(true)
```
Comment (andrewgarner) [inline spec/features/edit_dataset_spec.rb]: This seems unrelated to your commit?

Comment (benthorner) [reply spec/features/edit_dataset_spec.rb]: It's duplicated from the spec_helper.

Comment (benthorner) [reply spec/features/edit_dataset_spec.rb]: I don't think anyone's going to miss it ;-)
